server/.npmignore

```diff
@@ -0,0 +1,10 @@
+/*
```

Comment:
npmignore looks great :)
Maybe you need to remove the .ts files if not already.

Reply:
`src/*.ts` should be ignored by `/*`.